lib/java/test/org/apache/thrift/partial/ExceptionAsserts.java

```diff
@@ -84,6 +85,10 @@ public static <E extends Exception> void assertThrows(
     }
   }
 
+  /**
+   * @deprecated use {@link org.junit.jupiter.api.Assertions#assertThrows(Class, Executable)}
+   */
+  @Deprecated
```

Comment:
Why not just delete these helper methods? They probably shouldn't be used at all anymore.

Reply:
removed @ctubbsii